src/VirtualDevice.ts

```diff
@@ -168,7 +168,7 @@ export class VirtualDevice {
         let path = "/batch_process?";
         for (const key of Object.keys(this.configuration)) {
             const parameterValue = this.configuration[key];
-            if (!parameterValue) {
+            if (parameterValue === undefined) {
```

Comment:
what it about null value?

Reply:
Good question, though in the case of our configurations, I don't think we ever use null